canary/package.json

```diff
@@ -7,6 +7,9 @@
     "build": "yarn react && yarn vue && yarn angular",
     "pull": "chmod +x ./environments/pull-environments.sh && ./environments/pull-environments.sh",
     "install": "yarn react:install && yarn vue:install && yarn angular:install",
+    "script:install": "yarn --cwd script",
+    "updatePackage": "node script/changeCanaryPackageVersion.mjs",
+    "setup:next": "rm ../postcss.config.js && yarn script:install && yarn updatePackage",
```

Comment:
Do we also want to be able to run these commands from the root directory? Or are we ok with only running them from the `canary` dir?

Reply:
I'm okay with only running them from `canary/`, because this is an entirely different workflow from other directories.